e2e-tests/insurance/cypress/e2e/journeys/your-buyer/trading-history/trading-history-page.spec.js

```diff
@@ -67,91 +63,11 @@ context(
-          yesRadio().input().first().should('exist');
-
-          cy.checkText(yesRadio().label().first(), FIELD_VALUES.YES);
-        });
-
-        it('renders `no` radio button', () => {
-          noRadio().input().first().should('exist');
-
-          cy.checkText(noRadio().label().first(), FIELD_VALUES.NO);
-        });
-      });
-
-      describe(`${TOTAL_AMOUNT_OVERDUE} and ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
-        describe(`when not selecting a ${OUTSTANDING_PAYMENTS} radio`, () => {
-          it('should not render a heading', () => {
-            field(TOTAL_OUTSTANDING_PAYMENTS).heading().should('not.be.visible');
-          });
-
-          it(`should NOT render a label for ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
-            field(TOTAL_OUTSTANDING_PAYMENTS).label().should('not.be.visible');
-          });
-
-          it(`should NOT render an input for ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
-            field(TOTAL_OUTSTANDING_PAYMENTS).input().should('not.be.visible');
-          });
-
-          it(`should NOT render a label for ${TOTAL_AMOUNT_OVERDUE}`, () => {
-            field(TOTAL_AMOUNT_OVERDUE).label().should('not.be.visible');
-          });
-
-          it(`should NOT render an input for ${TOTAL_AMOUNT_OVERDUE}`, () => {
-            field(TOTAL_AMOUNT_OVERDUE).input().should('not.be.visible');
-          });
-        });
-
-        describe(`when clicking the 'yes' ${OUTSTANDING_PAYMENTS} radio`, () => {
-          beforeEach(() => {
-            cy.clickYesRadioInput();
-          });
-
-          it('should render a heading', () => {
-            cy.checkText(field(TOTAL_OUTSTANDING_PAYMENTS).heading(), FIELDS[TOTAL_OUTSTANDING_PAYMENTS].HEADING);
-          });
-
-          it(`should render a label for ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
-            cy.checkText(field(TOTAL_OUTSTANDING_PAYMENTS).label(), FIELDS[TOTAL_OUTSTANDING_PAYMENTS].LABEL);
-          });
-
-          it(`should render an input for ${TOTAL_OUTSTANDING_PAYMENTS}`, () => {
-            field(TOTAL_OUTSTANDING_PAYMENTS).input().should('be.visible');
-          });
-
-          it(`should render a label for ${TOTAL_AMOUNT_OVERDUE}`, () => {
-            cy.checkText(field(TOTAL_AMOUNT_OVERDUE).label(), FIELDS[TOTAL_AMOUNT_OVERDUE].LABEL);
-          });
-
-          it(`should render an input for ${TOTAL_AMOUNT_OVERDUE}`, () => {
-            field(TOTAL_AMOUNT_OVERDUE).input().should('be.visible');
-          });
-
-          it('should render a hyperlink for changing the currency', () => {
-            cy.checkLink(partials.provideAlternativeCurrencyLink(), alternativeCurrencyUrl, CONTENT_STRINGS.PROVIDE_ALTERNATIVE_CURRENCY);
-          });
-        });
-      });
-
-      describe(FAILED_PAYMENTS, () => {
-        const FIELD_ID = FAILED_PAYMENTS;
-
-        it('renders a label', () => {
-          cy.checkText(field(FIELD_ID).legend(), FIELDS[FIELD_ID].LABEL);
-        });
-
         it('renders `yes` radio button', () => {
```

Comment:
Do we need to add a test for the order - are they horizontal ordered? I'm not sure without looking at the design.

```
it('renders `yes` and `no` radio buttons in the correct order', () => {
  cy.assertYesNoRadiosOrder({ noRadioFirst: true });
});
```